tests/unit/core.js

```diff
@@ -1438,7 +1438,7 @@ exports.testDefaultArguments = function (test) {
     .addError(14, "'bar' is not defined.")
     .addError(14, "'num3' was used before it was declared, which is illegal for 'param' variables.")
     .addError(15, "'num4' was used before it was declared, which is illegal for 'param' variables.")
-    .addError(18, "Regular parameters cannot come after default parameters.")
+    .addError(11, "Regular parameters should not come after default parameters.")
```

Comment:
I have no idea how that happened - the tests fail as well because of this. I've pushed a commit to fix it. Thanks.